parts/k8s/containeraddons/1.17/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -338,7 +305,6 @@ spec:
     metadata:
       annotations:
         agentVersion: {{ContainerConfig "omsAgentVersion"}}
-        cluster-autoscaler.kubernetes.io/daemonset-pod: "true"
```

Comment:
Is there any reason to delete this? Looks like this has been added by the AKS-Engine team?